lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -594,7 +594,7 @@ def add_verification_value(xml, credit_card)
         #   Null-fill this attribute OR
         #   Do not submit the attribute at all.
         # - http://download.chasepaymentech.com/docs/orbital/orbital_gateway_xml_specification.pdf
-        xml.tag! :CardSecValInd, '1' if %w(visa discover diners_club).include?(credit_card.brand) && bin == '000001'
+        xml.tag! :CardSecValInd, '1' if %w(visa discover diners_club).include?(credit_card.brand)
```

Comment:
💬 Comment:

While checking the docs the only thing that seems odd, is that API docs version 9 says that CardSecValInd should be included when transacting with `visa`, `discover`, or `jbc` but here we have dinners_club.

![Orbital_Gateway_XML_Interface_Developer_Guide](https://user-images.githubusercontent.com/133580/222539222-aa0d4501-642f-47c5-80b4-e37d1cf5777d.png)



Should we update this section ?

Reply:
This is a good call out. I think for this change, we'll leave it as is and raise this internally.